operations/node_management/Add_a_Standard_Rack_Node.md

```diff
@@ -357,6 +357,8 @@ Follow the [Added Hardware](../network/management_network/added_hardware.md) pro
     ping "${BMC_XNAME}"
     ```
 
+1. If adding `Foxconn` (Paradise) nodes to the system, follow the [Replacing `Foxconn` Usernames and Passwords in Vault](./Replacing_Foxconn_User_Pass.md) procedure if nodes fail to discover.
```

Comment:
Same here, can remove ` around Foxconn if you add Foxconn to .spelling in the root directory.

Reply:
Added to dictionary